build/android/build.gradle

```diff
@@ -20,106 +25,110 @@ def sqlite3_version = "3240000"
 apply plugin: "com.android.application"
 
 android {
-	compileSdkVersion 26
-	buildToolsVersion "26.0.3"
-
-	defaultConfig {
-		versionCode 20
-		versionName "${System.env.VERSION_STR}.${versionCode}"
-		minSdkVersion 14
-		targetSdkVersion 14
-		applicationId "net.minetest.minetest"
-		manifestPlaceholders = [ package: "net.minetest.minetest", project: project.name ]
-		ndk {
-			// Specifies the ABI configurations of your native
-			// libraries Gradle should build and package with your APK.
-			abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a',
-			'arm64-v8a'
-		}
-	}
-
-	lintOptions {
-		disable "OldTargetApi", "GoogleAppIndexingWarning"
-	}
-
-	Properties props = new Properties()
-	props.load(new FileInputStream(file("local.properties")))
-
-	if (props.getProperty("keystore") != null) {
-		signingConfigs {
-			release {
-				storeFile file(props["keystore"])
-				storePassword props["keystore.password"]
-				keyAlias props["key"]
-				keyPassword props["key.password"]
-			}
-		}
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
+
+    defaultConfig {
+        versionCode 20
+        versionName "${System.env.VERSION_STR}.${versionCode}"
+        minSdkVersion 14
+        targetSdkVersion 28
+        applicationId "net.minetest.minetest"
+        manifestPlaceholders = [package: "net.minetest.minetest", project: project.name]
+        ndk {
+            // Specifies the ABI configurations of your native
+            // libraries Gradle should build and package with your APK.
+            // abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
+            abiFilters 'armeabi-v7a', 'x86'
```

Comment:
Sorry but we must keep all architectures.

Reply:
If you think so, check out the google documentation or run your own tests.

Reply:
@nerzhul happy about this?

Reply:
@nerzhul ?